src/scss/base/_global.scss

```diff
@@ -31,7 +31,7 @@ hr {
 
 a {
   color: $color-links;
-  &:hover {
+  &:hover:not(.btn--link):not(.btn--ghost) {
```

Comment:
Not sure about this. We don't want to couple classes to states in this way I don't think. Would be better overiding the attributes in each class. Could make maintaining difficult if classes are spread around different files.

Reply:
ok